examples/create-react-app-with-typescript/src/index.tsx

```diff
@@ -2,4 +2,4 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import Index from './pages/index';
 
-ReactDOM.render(<Index />, document.querySelector('#root'));
+ReactDOM.render(<Index projectName="Hello, World!" />, document.querySelector('#root'));
```

Comment:
I think that providing a property here is an edge case. Do we need to complexify the example?

Reply:
A component with props is one of the core use cases of components, no? Especially since this example should be used for issues I think that we should have a proper scaffold for props usage.

So for me this is the most basic usage of components and not some complex example. I would even argue that state and event handling (which is used in the example) is far more complex than simple props.

Providing props at this point is uncommon, yes. We could refactor this to have a dumb wrapper around index but I can't see any benefit.